packages/material-ui-icons/package.json

```diff
@@ -22,8 +22,9 @@
     "react-component"
   ],
   "scripts": {
-    "build": "npm run build:icons && npm run build:babel && npm run build:copy-files",
+    "build": "npm run build:icons && npm run build:babel && npm run build:babel:es2015module && npm run build:copy-files",
     "build:babel": "babel --presets es2015,react -d ./build ./src",
+    "build:babel:es2015module": "BABEL_ENV=modules babel ./src/index.js --presets=../../../scripts/material-ui-babel-preset-es2015 --out-file ./build/index.es.js",
```

Comment:
Why would we need to list all of the babel plugins in the 'material-ui-icons' package development dependencies? People need to install the root dependencies beforehand.

I would rather have things as symetric as possible.